app/src/main/res/xml/search_preferences.xml

```diff
@@ -28,42 +27,36 @@
         <SwitchPreference
             android:defaultValue="true"
             android:key="@string/pref_key_show_search_shortcuts"
-            android:title="@string/preferences_show_search_shortcuts"
-            app:iconSpaceReserved="false"/>
+            android:title="@string/preferences_show_search_shortcuts" />
         <SwitchPreference
             android:defaultValue="true"
             android:key="@string/pref_key_show_clipboard_suggestions"
-            android:title="@string/preferences_show_clipboard_suggestions"
-            app:iconSpaceReserved="false" />
+            android:title="@string/preferences_show_clipboard_suggestions" />
         <SwitchPreference
             android:defaultValue="true"
             android:key="@string/pref_key_search_browsing_history"
-            android:title='@string/preferences_search_browsing_history'
-            app:iconSpaceReserved="false" />
+            android:title='@string/preferences_search_browsing_history' />
         <SwitchPreference
             android:defaultValue="true"
             android:key="@string/pref_key_search_bookmarks"
-            android:title='@string/preferences_search_bookmarks'
-            app:iconSpaceReserved="false" />
+            android:title='@string/preferences_search_bookmarks' />
         <SwitchPreference
             android:defaultValue="true"
             android:key="@string/pref_key_show_voice_search"
             android:title="@string/preferences_show_voice_search"
-            app:iconSpaceReserved="false"
-            app:allowDividerAbove="false"/>
+            app:allowDividerAbove="false" />
         <SwitchPreference
             android:defaultValue="true"
             android:key="@string/pref_key_show_search_suggestions"
             android:title="@string/preferences_show_search_suggestions"
-            app:iconSpaceReserved="false"
-            app:allowDividerAbove="false"/>
+            app:allowDividerAbove="false" />
         <CheckBoxPreference
             android:defaultValue="false"
             android:dependency="@string/pref_key_show_search_suggestions"
             android:key="@string/pref_key_show_search_suggestions_in_private"
             android:title="@string/preferences_show_search_suggestions_in_private"
-            android:layout="@layout/checkbox_left_preference"
+            android:layout="@layout/checkbox_left_sub_preference"
```

Comment:
Let's leave precise checkbox alignment for another PR as you said, and just use icon space here

Reply:
@NotWoods using icon space takes icon to the right. It's on the left in ux mock. Is there a way to do this?

Reply:
If i use checkbox_left_preference icon is aligned to back arrow, not top bar. So looks like i must use checkbox_left_sub_preference. What do you think?

Reply:
My bad, I didn't catch that the icon space was already reserved.

From what I can tell, the checkbox is already in the correct position and doesn't need to be moved. The design changes focus on the search engine icons and headers.